ibm/service/vpc/data_source_ibm_is_lbs.go

```diff
@@ -56,6 +56,11 @@ func DataSourceIBMISLBS() *schema.Resource {
 							Computed:    true,
 							Description: "Load Balancer name",
 						},
+						isLBUdpSupported: {
```

Comment:
isLBUdpSupported is applicable only for network load balancer, hence i think optional: true should also be there, Please reverify